tests/test_1_new.ok

```diff
@@ -42,15 +42,15 @@ $ dit -v -d ./ditdir new .bo/bar -: 'fails: invalid group name'
 Error: Invalid group name: .bo
 Type 'dit --help' for usage details.
 ---------------------------------------------------
-$ find ./ditdir -type f
+$ ./list
```

Comment:
`./step` does now show anything after the first pipe. It will show `find ./ditdir -type f` instead of `find ./ditdir -type f | sort`, which is wrong.